config/opal_setup_wrappers.m4

```diff
@@ -142,13 +142,9 @@ AC_DEFUN([OPAL_SETUP_WRAPPER_INIT],[
           [AC_MSG_ERROR([--enable-wrapper-runpath cannot be selected with --disable-wrapper-rpath])])
 ])
 
-# Check to see whether the linker supports DT_RPATH.  We'll need to
-# use config.rpath to find the flags that it needs, if it does (see
-# comments in config.rpath for an explanation of where it came from).
-AC_DEFUN([OPAL_SETUP_RPATH],[
-    OPAL_VAR_SCOPE_PUSH([rpath_libdir_save rpath_script rpath_outfile])
-    AC_MSG_CHECKING([if linker supports RPATH])
-    # Output goes into globally-visible $rpath_args.  Run this in a
+AC_DEFUN([OPAL_EVAL_LIBTOOL],[
```

Comment:
Super minor nit pick: can you put a list of the input params as a comment above OPAL_EVAL_LIBTOOL?  I only mention this because it takes 4 params, and you'd have to dig through the m4 to figure out what they are.